assets/directory_asset_bundle.cc

```diff
@@ -31,7 +58,10 @@ bool DirectoryAssetBundle::GetAsBuffer(const std::string& asset_name,
 DirectoryAssetBundle::~DirectoryAssetBundle() {}
 
 DirectoryAssetBundle::DirectoryAssetBundle(std::string directory)
-    : directory_(std::move(directory)) {}
+    : directory_(std::move(directory)), fd_() {}
+
+DirectoryAssetBundle::DirectoryAssetBundle(fxl::UniqueFD fd)
+    : directory_(""), fd_(std::move(fd)) {}
```

Comment:
No need for ("").  The default constructor for std::string makes an empty string.